code/modules/antagonists/cult/team_cult.dm

```diff
@@ -165,36 +171,36 @@ RESTRICT_TYPE(/datum/team/cult)
 		cult_ascend()
 
 /datum/team/cult/proc/cult_rise()
-	cult_risen = TRUE
```

Comment:
This is the only place this is set to true. Wouldn't it screw with the check here? Meaning no cultist converted while the team has ascended will get the halo etc.

![image](https://github.com/ParadiseSS13/Paradise/assets/42044220/23e3253f-3f42-429e-946b-82605c0629b3)

Reply:
Its changed in a varset callback, (discussed on discord)